resource_type.go

```diff
@@ -188,6 +216,15 @@ func (t ResourceType) validateOperationValue(op PatchOperation) *errors.ScimErro
 		mapValue = map[string]interface{}{op.Path: op.Value}
 	}
 
+	// Check if it's a patch on a common attribute.
```

Comment:
Comment is a bit misleading since you first check that `op.Path` is equal to `externalId`.

Reply:
I've switched this by adding a SimpleCoreAttribute ExternalID to the schema on the fly